src/mpriscontroller.cpp

```diff
@@ -187,11 +247,11 @@ bool MprisController::openUri(const QUrl &uri)
     QStringList mimeNames = mime.aliases();
     mimeNames.prepend(mime.name());
     for (int i = 0; i < mimeNames.size(); i++) {
-        if (m_mprisRootInterface->supportedMimeTypes().contains(mimeNames[i])) {
-            QDBusPendingReply<> reply = m_mprisPlayerInterface->OpenUri(uri.toString());
+        if (priv->m_mprisRootInterface.supportedMimeTypes().contains(mimeNames[i])) {
```

Comment:
Old thing too, but that mimetype guessing and filtering seems pointless btw. The client can surely figure out itself if it's able to open something.